src/xdocs/config_imports.xml

```diff
@@ -997,6 +997,64 @@ public class InputIllegalImport { }
         </source>
 
         <p>
+          In the next examples usage of <code>strategyOnMismatch</code> property is shown.
+          This property defines strategy in a case when no matching allow/disallow rule was found.
+          <code>strategyOnMismatch</code> property is attribute of <code>import-control</code> and <code>subpackage</code> tags.
+          Property can have following values for <code>import-control</code> tag:
+          <ul>
+            <li>disallowed (default value) - if there is no matching allow/disallow rule in any of the subpackages, including the root level (import-control), then the import is allowed.</li>
+            <li>allowed - if there is no matching allow/disallow rule in any of the subpackages, including the root level, then the import is disallowed.</li>
+          </ul>
+          And following values for <code>subpackage</code> tags:
+          <ul>
+            <li>delegateToParent (default value) - if there is no matching allow/disallow rule inside the current subpackage, then it continues checking in the parent subpackage.</li>
+            <li>allowed - if there is no matching allow/disallow rule inside the current subpackage, then the import is allowed.</li>
+            <li>disallowed - if there is no matching allow/disallow rule inside the current subpackage, then the import is disallowed.</li>
+          </ul>
+        </p>
+
+        <p>
+          The following example demonstrates usage of <code>strategyOnMismatch</code> property for <code>import-control</code> tag. Here all imports are allowed except <code>java.awt.Image</code> and <code>java.io.File</code> classes.
+        </p>
+        <source>
+&lt;import-control pkg=&quot;com.puppycrawl.tools.checkstyle.checks&quot; strategyOnMismatch=&quot;allowed&quot;&gt;
+    &lt;disallow class=&quot;java.awt.Image&quot;/&gt;
+    &lt;disallow class=&quot;java.io.File&quot;/&gt;
+&lt;/import-control&gt;
+        </source>
+
+        <p>
+          In the example below, any import is disallowed inside <code>com.puppycrawl.tools.checkstyle.checks.imports</code> package except imports from package <code>javax.swing</code> and class <code>java.io.File</code>.
+          However, any import is allowed in the classes outside of <code>com.puppycrawl.tools.checkstyle.checks.imports</code> package.
+        </p>
+        <source>
+&lt;import-control pkg=&quot;com.puppycrawl.tools.checkstyle.checks&quot; strategyOnMismatch=&quot;allowed&quot;&gt;
+    &lt;subpackage name=&quot;imports&quot; strategyOnMismatch=&quot;disallowed&quot;&gt;
+        &lt;allow pkg=&quot;javax.swing&quot;/&gt;
+        &lt;allow class=&quot;java.io.File&quot;/&gt;
+    &lt;/subpackage&gt;
+&lt;/import-control&gt;
+        </source>
+
+        <p>
+          When <code>strategyOnMismatch</code> has <b>allowed</b> or <b>disallowed</b> value for <code>subpackage</code> tag, it makes <code>subpackage</code> isolated from parent rules.
```

Comment:
"<b>allowed</b> or <b>disallowed</b>" looks weird in html, please convert them to "code" as we use in all other cases "code" for values.